lib/mediaType.js

```diff
@@ -99,7 +99,7 @@ function specify(type, spec) {
 
   return {
     q: spec.q,
-    i: spec.i,
+    i: order,
```

Comment:
This may actually be a non-issue, I'm not sure. My comment was just from seeing the variable's value change to something else; I also haven't done any testing on this.